mod_reforged/hooks/config/tactical_entity_common.nut

```diff
@@ -0,0 +1,31 @@
+// We overwrite the vanilla function to remove all damage multiplier and add the burning damage type
+::Const.Tactical.Common.onApplyFire = function( _tile, _entity )
```

Comment:
This `onApplyFire` change should be a separate `fix` commit because this was an oversight in the original implementation where I converted the racial skills to use MSU damage type system.